src/components/dataset/list.vue

```diff
@@ -0,0 +1,48 @@
+<!--
+Copyright 2022 ODK Central Developers
+See the NOTICE file at the top-level directory of this distribution and at
+https://github.com/getodk/central-frontend/blob/master/NOTICE.
+
+This file is part of ODK Central. It is subject to the license terms in
+the LICENSE file found in the top-level directory of this distribution and at
+https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+including this file, may be copied, modified, propagated, or distributed
+except according to the terms contained in the LICENSE file.
+-->
+<template>
+  <div>
+    <dataset-table/>
+    <loading :state="$store.getters.initiallyLoading(['datasets'])"/>
+  </div>
+</template>
+
+<script>
+import DatasetTable from './table.vue';
+import Loading from '../loading.vue';
+
+import { apiPaths } from '../../util/request';
+import { noop } from '../../util/util';
+
+export default {
+  name: 'DatasetList',
+  components: { DatasetTable, Loading },
+  props: {
+    projectId: {
+      type: String,
+      required: true
+    }
+  },
+  created() {
+    this.fetchData();
+  },
+  methods: {
+    fetchData() {
+      this.$store.dispatch('get', [{
+        key: 'datasets',
+        url: apiPaths.datasets(this.projectId),
+        resend: false
+      }]).catch(noop);
```

Comment:
For now, this doesn't request extended metadata. If we add a "latest entity" timestamp as extended metadata at some point, we can change this to request extended metadata.